modules/installation/proc-add-custom-app-config-file-ocp-operator.adoc

```diff
@@ -0,0 +1,95 @@
+
+The custom application configuration file contains a sensitive environment variable, named `BACKEND_SECRET`, which is a mandatory backend authentication key that {product-short} uses to reference an environment variable defined in an {ocp-short} Secret. You must create a custom Secret, named 'secrets-rhdh', and reference it in the {product-short} Custom Resource.
+
+[NOTE]
+====
+You are responsible for protecting your {product} installation from external and unauthorized access. Manage the backend authentication key like any other secret. Meet strong password requirements, do not expose it in any configuration files, and only inject it into configuration files as an environment variable.
+====
+
+.Prerequisites
+* You have an active {ocp-brand-name} account.
+* Your administrator has installed the {product} Operator in {ocp-short}. For more information, see link:{LinkAdminGuide}[Installing the {product} Operator].
+* You have created the {product} Custom Resource in {ocp-short}.
+
+.Procedure
+. From the *Developer* perspective, select the *ConfigMaps* tab.
+. Click *Create ConfigMap*.
+. Select the *YAML view* option in *Configure via* and use the following example as a base template to create a ConfigMap, such as `app-config-rhdh.yaml`:
++
+[source,yaml,subs="attributes+"]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: app-config-rhdh
+data:
+  "app-config-rhdh.yaml": |
+    app:
+      title: {product}
+      baseUrl: https://backstage-<CUSTOM_RESOURCE_NAME>-<NAMESPACE_NAME>.<OPENSHIFT_INGRESS_DOMAIN>
+    backend:
+      auth:
+        keys:
+          - secret: "${BACKEND_SECRET}"
+      baseUrl: https://backstage-<CUSTOM_RESOURCE_NAME>-<NAMESPACE_NAME>.<OPENSHIFT_INGRESS_DOMAIN>
+      cors:
+        origin: https://backstage-<CUSTOM_RESOURCE_NAME>-<NAMESPACE_NAME>.<OPENSHIFT_INGRESS_DOMAIN>
+----
++
+.. Use the mandatory backend authentication key for {product} to reference an environment variable defined in an {ocp-short} Secret.
+.. Set the external URL of your {product} instance in the `app.baseUrl`, `backend.baseUrl` and `backend.cors.origin` fields of the application configuration. By default, the URL is similar to the following example:
+`pass:c[https://backstage-<CUSTOM_RESOURCE_NAME>-<NAMESPACE_NAME>.<OPENSHIFT_INGRESS_DOMAIN>;]`.
++
+[NOTE]
+====
+You can use the `oc get ingresses.config/cluster -o jsonpath='{.spec.domain}'` command to display your ingress domain. If you want to use a different host or sub-domain, customize the Custom Resource `spec.application.route` field and adjust the application configuration accordingly.
+====
+
+. Click *Create*.
+. Select the *Secrets* tab.
```

Comment:
Same for all instances of tab - should these be "view" if they're clicked on from the same left nav as the Topology view?